mappings/net/minecraft/structure/Structure.mapping

```diff
@@ -6,11 +6,31 @@ CLASS net/minecraft/class_3499 net/minecraft/structure/Structure
 	METHOD method_15160 getSize ()Lnet/minecraft/class_2338;
 	METHOD method_15161 setAuthor (Ljava/lang/String;)V
 		ARG 1 name
+	METHOD method_15162 addRotatedAndMirroredOffset (Lnet/minecraft/class_2338;Lnet/minecraft/class_2415;Lnet/minecraft/class_2470;II)Lnet/minecraft/class_2338;
```

Comment:
maybe `map` or `transform`, and arg3 and arg 4 are like `sizeX` and `sizeZ`

Reply:
`map`/`transform` sound a bit too generic to my taste, maybe someone else has a though on that

Reply:
"mirror and rotate" is way too wordy. A structure can have a position remapped by a mirror and a rotation combination, so `map` or `remap` is accurate.

Reply:
@Runemoro want to hear from you here.

Reply:
The operation of mirroring, rotating, and offsetting is called an [isometry](https://en.m.wikipedia.org/wiki/Isometry). But I don't know how to turn that into a method name... `applyIsometry`? `isometricTransform`? `transform`?

Reply:
I might be wrong, but the actual code is about mirroring and rotating the given _offset_, and then applying it to the position, which is even trickier

Reply:
I might be wrong, but the actual code is about mirroring and rotating the given _offset_, and then applying it to the position, which is even trickier

Reply:
Then `applyTransformedOffset`. The specific transformations applied to the offset should just be in the parameter names.

Reply:
What’s about its derivative `addMirroredAndRotatedSize`, which uses structure size as offset; is it good as `offsetByTransformedSize`?

Reply:
Should rename that `offset ` to `pivot`

Reply:
In this case it is an offset, not a center of the rotation

Reply:
Hmm, `applyOffsetTransform`?

Reply:
It applies transformed offset, not offset(ted) transformation

Reply:
Then I support runemoro's `applyTransformedOffset`

Reply:
Already committed that